regression/strings-smoke-tests/java_parselong/test_parselong.java

```diff
@@ -0,0 +1,39 @@
+public class test_parselong
+{
+    public static void main(String[] args)
+    {
+        if (args.length == 1) {
```

Comment:
35 seconds for all 5? I think that is worth checking in, but probably not fast enough to switch them to CORE unfortunately.